tabpy-server/tabpy_server/common/tabpy_logging.py

```diff
@@ -1,8 +1,37 @@
-import simplejson
-class PYLogging(object): 
+import json as simplejson
+import logging
+from logging import handlers
+import tempfile
+import os
+
+if os.path.isfile('./common/config.py'):
+    import common.config as config
+    LOG_LEVEL = config.TABPY_LOG_LEVEL
+else:
+    LOG_LEVEL = "INFO"
+
+class PYLogging(object):
     @classmethod
     def initialize(cls, logger):
         cls.logger = logger
+        cls.logger.setLevel = logging.getLevelName(LOG_LEVEL)
+
+        # create formatter
+        formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')
+        temp_dir = tempfile.gettempdir()
+        fh_info = handlers.RotatingFileHandler(filename=os.path.join(temp_dir, "tabpy_info.log",),
+                                               maxBytes=10000000, backupCount=5)
+        fh_info.setLevel(logging.INFO)
```

Comment:
Just to confirm - additionally to default logging with configurable log level with this change we are going to have log files with info and debug level in tmp folder?

Reply:
@evaliula Why do we need both - can we keep just the most verbose? Also, what is the purpose of user-defined logging level if we still to log with DEBUG/INFO?

Reply:
we can keep the most verbose one, I separate them because debug might have too much trace and it just a convenience if a parser is written or verification is based on INFO. If user only define logging info level , debug log file will not log debug trace.